usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -197,9 +198,15 @@ typedef struct {
 
 #define MAX_RETRY_COUNT 100
 
-#define RETRY_START             do {                                     \
+#define RETRY_START(rc, tokdata)                                         \
+                                do {                                     \
                                     int retry_count;                     \
+                                    ep11_target_info_t* target_info =    \
+                                               get_target_info(tokdata); \
```

Comment:
Hm, well, it would also not compile if there are parenthesis around and tokdata expands to something that's invalid. For such single parameter lists parenthesis will not help anyway.

Reply:
You could have a statement involving the comma operator as tokdata, but not if put like this.  So typical C convention would be to add an additional pair of parenthesis.